### PR TITLE
Issue 5 - episode number in URL

### PR DESCRIPTION
**Pull request by DvdQzd**
Solves https://github.com/jilleJr/musicforprogramming.ipfs/issues/5

### CODE REVIEW

File: src/App.svelte [hunk 1]
```diff
@@ -31,9 +32,17 @@
     }
   }
 
+  function getEpisodeNumberFromQueryParams(): string {
+    const episodeNumber = queryString.parse(location.search).episode;
+    return Array.isArray(episodeNumber) ? episodeNumber[0] : episodeNumber;
```
Comment (jilleJr) [inline src/App.svelte:37]: The use of this new package `query-string` seems unmotivated when we're already using `URLSearchParams`. We're parsing the query string twice here.

Would like if you used the [`searchParams.get()`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get) here, for example:

```suggestion
    return searchParams.get("episode");
```

And then removing the dependency on `query-string`.

**Review by jilleJr — CHANGES_REQUESTED** [review]
Congratulations, you are the first contributor to this repo :tada: :heart: 

Could you remove the dependency on `query-string`?

File: src/App.svelte (hunk 1 above)
Comment (DvdQzd) [reply src/App.svelte:37]: You're right! 

On it.

Comment (DvdQzd) [reply src/App.svelte]: Done!

**Review by jilleJr — APPROVED** [review]
Awesome PR! Small and concise

**Comment by jilleJr** [comment]
Thank you very much for contributing!

**Comment by jilleJr** [comment]
Hey @DvdQzd, I forgot to license this repo. I wish to license this under GNU GPL v3, and by obligation I have to ask you if you're OK with that as your code is in the repo now.

Are you fine by licensing your contribution under GNU GPL v3?

**Comment by DvdQzd** [comment]
Sure. No problem!

**Pull request merged**